Authorization Server deprecation (#5020)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/concepts/key-rotation/index.md

> **Note:** The current Okta key rotation schedule is four times a year, but can change without notice. New keys are normally generated a few weeks before the rotation occurs to ensure that downstream customer caching mechanisms are updated before the rotation occurs.

> If you are using a custom authorization server, configure and perform key rollover/rotation at the [Authorization Server level](https://developer.okta.com/docs/api/openapi/okta-management/management/tag/AuthorizationServerKeys/#tag/AuthorizationServerKeys/operation/rotateAuthorizationServerKeys).

Reviewer: ```suggestion
If you're using a custom authorization server, configure and perform key rollover/rotation at the [Authorization Server level](https://developer.okta.com/docs/api/openapi/okta-management/management/tag/AuthorizationServerKeys/#tag/AuthorizationServerKeys/operation/rotateAuthorizationServerKeys).
```

> If you are using the org authorization server, configure and perform key rollover/rotation at the [client level](/docs/reference/api/apps/#generate-new-application-key-credential).

Reviewer: ```suggestion
If you're using the org authorization server, configure and perform key rollover/rotation at the [client level](/docs/reference/api/apps/#generate-new-application-key-credential).
```

* To save the network round trip, cache the `jwks_uri` response locally with respect to the directives in the [standard HTTP Cache-Control headers](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Cache-Control). The cache-control directives are relative to the time of the request. If you make a request as the safe cache period ends, Okta returns the `no-cache` directive to ensure that you don't cache keys that will soon expire.

> * You can switch the authorization server key rotation mode by updating the authorization server's `rotationMode` property. For more information see the API Reference: [Authorization Server Credentials Signing Object](https://developer.okta.com/docs/api/openapi/okta-management/management/tag/AuthorizationServerKeys/#tag/AuthorizationServerKeys/operation/rotateAuthorizationServerKeys).

Reviewer: ```suggestion
* You can switch the authorization server key rotation mode by updating the authorization server's `rotationMode` property. For more information, see the API Reference: [Authorization Server Credentials Signing Object](https://developer.okta.com/docs/api/openapi/okta-management/management/tag/AuthorizationServerKeys/#tag/AuthorizationServerKeys/operation/rotateAuthorizationServerKeys).
```

> > **Caution:** Keys used to sign tokens automatically rotate and should always be resolved dynamically against the published JWKS. Your app might fail if you hardcode public keys in your applications. Be sure to include key rollover in your implementation.

Reviewer: ```suggestion
> **Caution:** Keys used to sign tokens automatically rotate and should always be resolved dynamically against the published JWKS. Your app might fail if you hardcode public keys. Be sure to include key rollover in your implementation.
```
